library/src/plugins/official/logic/actions/setAll.ts

```diff
@@ -10,7 +11,7 @@ export const SetAll: ActionPlugin = {
   fn: ({ signals }, prefix: string, newValue) => {
     signals.walk((path, signal) => {
       if (!path.startsWith(prefix)) return
-      signal.value = newValue
+      ;(signal as Signal<any>).value = newValue
```

Comment:
Redundant semicolon?

Reply:
Oh my sweet summer child. JS wat my dear

Reply:
🖕JS. Could we rewrite to the following for clarity?
```js
if (!path.startsWith(prefix)) {
  return
}
```

Reply:
Done.